PowerEditor/src/WinControls/DockingWnd/DockingCont.cpp

```diff
@@ -952,18 +974,18 @@ LRESULT DockingCont::runProcTab(HWND hwnd, UINT Message, WPARAM wParam, LPARAM l
 			tcItem.mask		= TCIF_PARAM;
 			::SendMessage(hwnd, TCM_GETITEM, iItem, reinterpret_cast<LPARAM>(&tcItem));
 			if (!tcItem.lParam)
-				return FALSE;
+				return 1;

```

Comment:
IMO it should return 0 anyway.

Reply:
This should be `break;` instead, to pass it to default window processing.